Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -386,8 +386,12 @@ function Get-ApplicationEndpoint([string] $appName, [string] $applicationEndpoin
             $defaultAppEndpoint = $userEnteredApplicationEndpoint
         }
     }
-    if($defaultAppEndpoint){ Add-InstallationSetting $scope "applicationEndPoint" "$defaultAppEndpoint" $installConfigPath | Out-Null }
-    if($defaultAppEndpoint){ Add-InstallationSetting "common" "$($scope)Service" "$defaultAppEndpoint" $installConfigPath | Out-Null }
+
+    # Do not need to add IdPSS to common config, only used by Identity
+    if($defaultAppEndpoint -notlike "*IdentityProviderSearchService*"){
```

Comment:
Is this string in a constant somewhere?

Reply:
We should move this up as a global variable, or even better make it a parameter with a default value of "IdenitytProviderSearchService".

Reply:
Make IdentityProviderSearchService a variable - even better if you can make it a parameter with the default value set to "IdentityProviderSearchService", but allow overrides.

Reply:
maybe a parameter/flag it check against before writing to the install.config instead?

Reply:
does idpss still need to have the variable applicationEndpoint created for the Idpss scope. if($defaultAppEndpoint){ Add-InstallationSetting $scope "applicationEndPoint" "$defaultAppEndpoint" $installConfigPath | Out-Null }

Reply:
Let me verify, but I'm pretty sure nothing uses it, it's just in there because we're reusing the identity Get-ApplicationEndpoint which adds it in by default

Reply:
Ya nothing references this anywhere from what I can see

Reply:
ok.